integration/fixture_test.go

```diff
@@ -1,5 +1,3 @@
-//+build integration
```

Comment:
Did you mean to remove this?

Reply:
yup! it makes the tooling check this code a bit better when the tags aren't on